tools/build-cocoa.sh

```diff
@@ -235,23 +289,30 @@ if [[ -n $COPY ]]; then
     cp -R core "${DESTINATION}"
 
     if [[ ! -z $BUILD_XCFRAMEWORK ]]; then
-        rm -rf "${DESTINATION}/realm-sync.xcframework"
-        cp -Rc realm-sync.xcframework "${DESTINATION}"
-        rm -rf "${DESTINATION}/realm-sync-dbg.xcframework"
-        cp -Rc realm-sync-dbg.xcframework "${DESTINATION}"
+        rm -rf "${DESTINATION}/realm-monorepo.xcframework"
+        cp -Rc realm-monorepo.xcframework "${DESTINATION}"
+        rm -rf "${DESTINATION}/realm-monorepo-dbg.xcframework"
+        cp -Rc realm-monorepo-dbg.xcframework "${DESTINATION}"
     fi
 else
-    rm -f "realm-core-cocoa-${VERSION}.tar.xz"
-    rm -f "realm-parser-cocoa-${VERSION}.tar.xz"
+    rm -f "realm-monorepo-cocoa-${VERSION}.tar.xz"
     # .tar.gz package is used by realm-js, which uses the parser
-    tar -czvf "realm-core-cocoa-${VERSION}.tar.gz" core
+    tar -czvf "realm-monorepo-cocoa-${VERSION}.tar.gz" --exclude "realm-monorepo*.xcframework" core
     # .tar.xz package is used by cocoa, which doesn't use the parser
-    tar -cJvf "realm-core-cocoa-${VERSION}.tar.xz" --exclude "realm-parser*.xcframework" core
-    tar -cJvf "realm-parser-cocoa-${VERSION}.tar.xz" core/realm-parser*.xcframework
+    tar -cJvf "realm-monorepo-cocoa-${VERSION}.tar.xz" --exclude "realm-monorepo*.xcframework" core
 
     if [[ ! -z $BUILD_XCFRAMEWORK ]]; then
-        rm -f "realm-sync-xcframework-${VERSION}.tar.xz"
+        rm -f "realm-parser-cocoa-${VERSION}.tar.xz"
+        tar -cJvf "realm-parser-cocoa-${VERSION}.tar.xz" core/realm-parser*.xcframework
+        rm -f "realm-monorepo-xcframework-${VERSION}.tar.xz"
         # until realmjs requires an xcframework, only package as .xz
-        tar -cJvf "realm-sync-xcframework-${VERSION}.tar.xz" realm-sync.xcframework realm-sync-dbg.xcframework
+        tar -cJvf "realm-monorepo-xcframework-${VERSION}.tar.xz" core/realm-monorepo.xcframework core/realm-monorepo-dbg.xcframework
+
+        # Swift Package Manager only supports a zip containing only the xcframework
```

Comment:
We can skip the spm package as that was just a temporary solution until we could adopt the monorepo.